windows/T1547.001.csv

```diff
@@ -0,0 +1,78 @@
+_E_,File,CREATE,path~=AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\jsestartup.jse
+_E_,File,CREATE,path~=ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp\jsestartup.jse
+_E_,Process,cmdline~=cscript.exe,cmdline~=AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\jsestartup.jse
+_E_,Process,cmdline~=cscript.exe,cmdline~=ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp\jsestartup.jse
+
+T1547.001,windows,5b6768e4,Suspicious bat file run from startup Folder
+_E_,File,CREATE,path~=AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\batstartup.bat
+_E_,File,CREATE,path~=ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp\batstartup.bat
+_E_,Process,cmdline~=cmd.exe,cmdline~=AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\batstartup.bat
+_E_,Process,cmdline~=cmd.exe,cmdline~=ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp\batstartup.bat
+
+T1547.001,windows,24e55612,Add Executable Shortcut Link to User Startup Folder
+_E_,File,CREATE,path~=AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\calc_exe.lnk
+
+T1547.001,windows,bda6a3d6,Add persistance via Recycle bin
+_E_,REG,event_type~=SETVALUEKEY,key_name=\REGISTRY\MACHINE\SOFTWARE\Classes\CLSID\{645FF040-5081-101B-9F08-00AA002F954E}\shell\open\command
+
+T1547.001,windows,9dc7767b,SystemBC Malware-as-a-Service Registry
+ARG,reg_key_value, -windowstyle hidden -ExecutionPolicy Bypass -File
+ARG,reg_key_path,HKCU:\Software\Microsoft\Windows\CurrentVersion\Run
+_E_,REG,event_type=SETVALUEKEY,key_name~=\REGISTRY\USER,key_name~=SOFTWARE\Microsoft\Windows\CurrentVersion\Run,value_name=socks5_powershell
+
+T1547.001,windows,acfef903,Change Startup Folder - HKLM Modify User Shell Folders Common Startup Value
+ARG,new_startup_folder,%TMP%\atomictest\
+ARG,payload,C:\Windows\System32\calc.exe
+_E_,REG,event_type=SETVALUEKEY,key_name=\REGISTRY\MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders,value_name=Common Startup
+
+T1547.001,windows,8834b65a,Change Startup Folder - HKCU Modify User Shell Folders Startup Value
+ARG,new_startup_folder,%TMP%\atomictest\
+ARG,payload,C:\Windows\System32\calc.exe
+_E_,REG,event_type=SETVALUEKEY,key_name~=\REGISTRY\USER,key_name~=SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders,value_name=Startup
+
+T1547.001,windows,a70faea1,HKCU - Policy Settings Explorer Run Key
+ARG,target_key_value_name,atomictest
+ARG,payload,C:\Windows\System32\calc.exe
+_E_,REG,event_type=SETVALUEKEY,key_name~=REGISTRY\USER,key_name~=SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer\Run,value_name=atomictest
+
+T1547.001,windows,b5c9a9bc,HKLM - Policy Settings Explorer Run Key
+ARG,target_key_value_name,atomictest
+ARG,payload,C:\Windows\System32\calc.exe
+_E_,REG,event_type=SETVALUEKEY,key_name=\REGISTRY\MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer\Run,value_name=atomictest
```

Comment:
Why not use the args ?
```suggestion
ARG,target_key_value_name,atomictest
ARG,payload,C:\Windows\System32\calc.exe
_E_,REG,event_type=SETVALUEKEY,key_name=\REGISTRY\MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer\Run,value_name=${target_key_value_name}
```

Reply:
Yup. That's a better alternative. Used #{var} format as specified by atomic red team.